shell/platform/windows/client_wrapper/include/flutter/flutter_view.h

```diff
@@ -20,8 +20,10 @@ class FlutterView {
   FlutterView(FlutterView const&) = delete;
   FlutterView& operator=(FlutterView const&) = delete;
 
+#ifndef WINUWP
   // Returns the backing HWND for the view.
   HWND GetNativeWindow() { return FlutterDesktopViewGetHWND(view_); }
+  #endif
```

Comment:
Should we go ahead and add the corresponding CoreWindow accessor?

Reply:
We can although not sure what the scenarios will be yet (one possible one is configuring the non-client area).  It is possible for plugins to get the current `CoreWindow` using `GetForCurrentThread` from the main thread so it may be redundant to expose.